### ISSUE TITLE
remove trailing whitespaces from drugbank.synonyms values

### ISSUE THREAD

**Issue by newgene** [biothings/mychem.info#160]
https://mychem.info/v1/query?q=unii.rxcui:68442&fields=drugbank.synonyms

```
            "drugbank": {
                "_license": "https://go.drugbank.com/releases/latest#open-data",
                "synonyms": [
                    "Filgrastim",
                    "Filgrastim-aafi",
                    "Filgrastim-sndz",
                    "G-CSF ",
                    "Granulocyte Colony Stimulating Factor ",
                    "Tbo-filgrastim"
                ]
            }
```

**Comment by colleenXu** [comment]
Note that I mention this field here in a potential annotation service effort https://github.com/biothings/biothings_explorer/issues/344#issuecomment-961647522

**Comment by DylanWelzel** [comment]
fixed with #172
